.github/workflows/ci.yaml

```diff
@@ -54,21 +54,63 @@ jobs:
         if: matrix.goos == 'linux' && matrix.goarch == 'amd64'
         run: make ci
 
+  chart-lint:
+    name: Helm chart Lint
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: helm/chart-testing-action@v2.2.1
+
+      - name: Lint charts
+        run: ct --config .github/ct.yaml lint --debug --all
+
+      - name: Check for changed installable charts
+        id: list-changed
+        run: |
+          changed=$(ct --config .github/ct.yaml list-changed)
+          if [[ -n "$changed" ]]; then
+            echo "::set-output name=changed::true"
+          fi
+
+      - name: Set up helm-unittest
+        if: steps.list-changed.outputs.changed == 'true'
+        run: |
+          helm plugin install https://github.com/quintush/helm-unittest
+      - name: Run unit tests
+        if: steps.list-changed.outputs.changed == 'true'
+        run: |
+          for chart in $(ct --config .github/ct.yaml list-changed); do
+            if [ -d "$chart/tests/" ]; then
+              helm unittest -3 $chart
+            fi
+          done
+
+      - name: Create kind cluster
+        uses: helm/kind-action@v1.2.0
+        if: steps.list-changed.outputs.changed == 'true'
+      - name: Create test ns
+        if: steps.list-changed.outputs.changed == 'true'
+        run: kubectl create ns test
+
+      - name: Test install charts
+        run: ct --config .github/ct.yaml install --namespace test
```

Comment:
Moreover I am not sure it worth to install it, since we to it again in the integration tests just below

Reply:
I agree with the need to install it, but then we need to build and use the image from main, otherwise I guess it would break if config changes from one of the two sides.

It would be the very same of having the chart in a separate repo if we do not compile main, no?

Reply:
Maybe you are right that I should move it. Let me rethink it a couple of minutes while I fix the pipeline...

Reply:
I thought about it and you are right. The installation test does test the latest version when the chart changes but the integration test also does it so it doesn't make sense to test it twice.

Removed in [#0264a4a](https://github.com/newrelic/newrelic-k8s-metrics-adapter/pull/40/commits/0264a4a81c1f8b0b8d7092c78cf6fac92d08ca3b)